cmd/root.go

```diff
@@ -7,30 +7,16 @@ import (
 )
 
 var rootCmd = &cobra.Command{
-	Use:   "hugo",
-	Short: "Hugo is a very fast static site generator",
-	Long: `A Fast and Flexible Static Site Generator built with
-                love by spf13 and friends in Go.
-                Complete documentation is available at http://hugo.spf13.com`,
+	Use:   "hwsc-dev-ops",
+	Short: "Humpback Whale Social Call command line tool",
+	Long:  "For more information, select following available commands",
```

Comment:
"For more information, select the following available commands",

Missing `the`